pkg/ingester/ingester.go

```diff
@@ -144,6 +147,8 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 	f.Int64Var(&cfg.DefaultLimits.MaxInflightPushRequests, "ingester.instance-limits.max-inflight-push-requests", 0, "Max inflight push requests that this ingester can handle (across all tenants). Additional requests will be rejected. 0 = unlimited.")
 
 	f.StringVar(&cfg.IgnoreSeriesLimitForMetricNames, "ingester.ignore-series-limit-for-metric-names", "", "Comma-separated list of metric names, for which -ingester.max-series-per-metric and -ingester.max-global-series-per-metric limits will be ignored. Does not affect max-series-per-user or max-global-series-per-metric limits.")
+
+	f.StringVar(&cfg.AdminContactDetails, "ingester.admin-contact", "", "Add site administrator contact details for customising error messages")
```

Comment:
```suggestion
	f.StringVar(&cfg.AdminLimitMessage, "ingester.admin-limit-message", defaultAdminLimitMessage, "Customize the message contained in limit errors")
```
Where defaultAdminLimitMessage is a string var with "please contact administrator to raise it" that can be used in tests